### DISCUSSION TITLE
copy badge display not proper

### DISCUSSION THREAD

**Discussion by pavan2108** [npm/feedback#690] [Website] [answered]
When ever we copy the command to install we get a message that the command is copied
![Screenshot from 2022-05-19 21-21-18](https://user-images.githubusercontent.com/68530522/169343525-f9710fb1-72f0-4e8f-a809-b0eb89571d9c.png)

But when I do the same by scrolling my windows a bit I can't see the message and the message can only be see after scrolling top to website

**Comment by monishcm** [comment, marked as answer]
Thanks for the feedback. Our design team will investigate this and update the backlog.
